### ISSUE TITLE
child components

### ISSUE THREAD

**Issue by suenot** [guillaumevincent/vue-polyglot#5]
How to use:
```js
Vue.locales({
  'en': {
    'title': 'Coins list'
  },
  'ru': {
    'title': 'Список монет'
  },
  'zh': {
    'title': '資產'
  }
})
```
in child ```coin.vue``` component? This overrides main comonent config ```app.vue```.

**Comment by guillaumevincent** [comment]
@suenot if I understand you properly, you want to be able to add locale directly inside your component?

**Comment by guillaumevincent** [comment]
See https://github.com/guillaumevincent/vue-polyglot#extend-translations-synchronously
Feature available in version 0.2.0
